### DISCUSSION TITLE
Add Startisback's colour menu and custom colour saving ability

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#192] [Ideas]
Startisback's colour picker has lots of preset colours and it lets you save up to 16 custom colours. It would be nice to have something like this in DWMBlurGlass.
![image](https://github.com/Maplespe/DWMBlurGlass/assets/104862146/f4179d2b-dcc0-402e-bf67-621bfcf29181)

**Comment by AllieTheFox** [comment]
> Startisback's colour picker has lots of preset colours and it lets you save up to 16 custom colours. It would be nice to have something like thisin DWMBlurGlass. ![image](https://private-user-images.githubusercontent.com/104862146/307284892-f4179d2b-dcc0-402e-bf67-621bfcf29181.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg3MTQ5NDAsIm5iZiI6MTcwODcxNDY0MCwicGF0aCI6Ii8xMDQ4NjIxNDYvMzA3Mjg0ODkyLWY0MTc5ZDJiLWRjYzAtNDAyZS1iZjY3LTYyMWJmY2YyOTE4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyM1QxODU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMDU1NDNjY2I2MDM2MmEwMGQzNGNhYWNiOWExY2M4ZjY2ZmVlMTg4ZmJlOGQ1YThlMTEwYmU2NTU5MjJhODIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RFn9Js9LjfIRAHf-L6QIwFqHen7Ss85y0PjZezuWHeI)

Yessssss
Was just about to make a feature request myself for this
